src/resourcesSpec.ts

```diff
@@ -82,36 +94,274 @@ export function getResourceTypeAttribute(type: string, attributeName: string): A
     return resourceAttribute
 }
 
-function getPropertyType(type: string){
-    const propertyType =  specification.PropertyTypes[type];
-    if (!propertyType) {
-        throw new NoSuchPropertyType(type);
+function getPropertyType(type: string): awsData.ResourcePropertyType {
+
+    // destructure property name
```

Comment:
could we have a comment with the examples of what is expected here? The reuse of variables isn't the clearest. 

Something like
`// ex. baseType = AWS::EC2::Instance, baseTypeName = ABCD, etc`

Will help someone (and me) understand the code easier

Reply:
Hopefully, it should be OK now...